artemis/src/main/java/tech/pegasys/artemis/cli/deposit/EncryptedKeystoreWriter.java

```diff
@@ -56,18 +56,18 @@ public void writeKeys(final BLSKeyPair validatorKey, final BLSKeyPair withdrawal
     final KeyStoreData withdrawalKeyStoreData =
         generateKeystoreData(withdrawalKey, withdrawalKeyPassword);
 
-    saveKeyStore(
-        keystoreDirectory.resolve("validator_" + validatorKey.getPublicKey().toString() + ".json"),
-        validatorKeyStoreData);
-    saveKeyStore(
-        keystoreDirectory.resolve(
-            "withdrawal_" + withdrawalKey.getPublicKey().toString() + ".json"),
-        withdrawalKeyStoreData);
+    final String validatorFileName =
+        "val_" + trimPublicKey(validatorKey.getPublicKey().toString()) + ".json";
+    final String withdrawalFileName =
+        "wdr_" + trimPublicKey(withdrawalKey.getPublicKey().toString()) + ".json";
```

Comment:
Why did these change to using abbreviations instead of the full `validator` and `withdrawal` prefix?

Reply:
I tried to keep it short, shall I restore the full prefix?

Reply:
Yeah unless there's some technical advantage to it being short I'd stick with the full prefix so it's easier for users to understand.